webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -4,40 +4,50 @@ import store from '../javascripts/react_app/redux';
 
 import * as ToastActions from './react_app/redux/actions/toasts';
 
+const notificationTypeToAlertType = (type) => {
+  switch (type) {
+    case 'notice':
+      return 'success';
+    default:
+      return type;
+  }
+};
+
 const isSticky = type => ['notice', 'success', 'info'].indexOf(type) === -1;
 
 export function notify({
   message, type, link, sticky = isSticky(type),
 }) {
   store.dispatch(ToastActions.addToast({
+    type: notificationTypeToAlertType(type),
     message,
-    type,
     sticky,
     link,
   }));
 }
 
-export function clear() {
+export const clear = () => {
   store.dispatch(ToastActions.clearToasts());
-}
+};
 
-function importFlashMessagesFromRails() {
+const importFlashMessagesFromRails = () => {
   const notifications = $('#notifications');
 
-  if (notifications.length === 0 ||
-    !notifications.data('flash')) { return; }
+  if (notifications.length === 0 || !notifications.data('flash')) {
+    return;
+  }
 
   notifications.data('flash').forEach(([type, message]) => {
     notify({ message, type });
   });
   // remove both jquery cache and dom entry to avoid ajax ContentLoad events
   // reloading our notifications
   notifications.attr('data-flash', '').removeData();
-}
+};
 
 // load notifications from Rails on ContentLoad
 // to accommodate rails flash syntax
-$(document).on('ContentLoad', () => {
+document.addEventListener('ContentLoad', () => {
```

Comment:
is this required? AFAIR it doesnt work on all IE versions?

Reply:
Not required, just something i thought i can change on the go.
It should work for IE >= 9
https://www.w3schools.com/jsref/met_document_addeventlistener.asp

Reply:
should we be consistent and apply it to the entire code base?

Reply:
could be a problem applying this to the entire code base, because we're using  jquery.turbolinks which fix  binded events problem caused by turbolinks.